recipe/conda_build_config.yaml

```diff
@@ -308,7 +308,7 @@ pin_run_as_build:
   readline:
     max_pin: x
   r-base:
-    max_pin: x.x.x
+    max_pin: x.x
```

Comment:
@mingwandroid, @conda-forge/r should this have a `min_pin: x.x` as well?

Reply:
I think so, pinging @msarahan and @katietz

Reply:
Just to aid my understanding of this:
Is `min_pin: x.x` added because `r-base` provides a runtime like `python` thus many packages depend on it? Hence the `min_pin: x.x` would avoid the need to update `r-base` in a Conda environment when installing a single package build against a new patch release.
And/or is it because it generally doesn't matter for R against which patch release you build a package (e.g., no linking taking place and/or patch releases guaranteed to be completely forward/backward compatible within a minor version release series)?

Reply:
I would be nice if we could document the reason for the `min_pin` (i.e., add a comment in `conda_build_config.yaml`) here since `min_pin` is rather uncommon our current setup.